src/components/phase/PhaseApp.js

```diff
@@ -122,6 +134,16 @@ class PhaseApp extends Component {
             return patchAppInstanceResource(payload, this.postMessage);
           case GET_APP_INSTANCE:
             return dispatchGetAppInstance(payload, this.postMessage);
+          case POST_ACTION: {
+            // add geolocation to action if enabled
+            if (geolocationEnabled) {
+              const {
+                coords: { latitude, longitude },
+              } = geolocation.toJS();
```

Comment:
Why are you using `toJS` and not accessing like an Immutable object (`geolocation.getIn...`)?

Reply:
I did it like in `components/SpacesNearby.js`, but I can change it.